editor/inserter/menu.js

```diff
@@ -267,14 +264,48 @@ class InserterMenu extends Component {
 		);
 	}
 
-	switchTab( tab ) {
-		this.setState( { tab: tab } );
+	renderBlocks( separatorSlug, blocks ) {
+		const { instanceId } = this.props;
+		return (
+			<div
+				className="editor-inserter__category-blocks"
+				role="menu"
+				tabIndex="0"
+				aria-labelledby={ `editor-inserter__separator-${ separatorSlug }-${ instanceId }` }
```

Comment:
Looking again, it's an issue with "embed" as well. A couple considerations:

- For both recent and embed, we pass `null` as first argument, and implementation of `renderBlocks` detects this in determining whether to assign `aria-labelledby`
- Optionally with above, treat first argument of `renderBlocks` as the ID of labelledby, only specified from `renderCategory`. Maybe this should be the second argument, so in render we can just call `this.renderBlocks( this.props.recentlyUsedBlocks )`